### PR TITLE
413 resolve light warnings for mylyn tasks

### PR DESCRIPTION
**Pull request by gnl42**

### CODE REVIEW

**Review by ruspl-afed — COMMENTED** [review]
Please follow [guidelines for copyright headers](https://www.eclipse.org/projects/handbook/#ip-copyright-headers) 

All the rest looks good, except red pipelines.

File: mylyn.tasks/org.eclipse.mylyn.tasks.bugs/src/org/eclipse/mylyn/internal/tasks/bugs/wizards/SelectSupportElementPage.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2004, 2012 Tasktop Technologies and others.
+ * Copyright © 2004, 2012, 2024 Tasktop Technologies and others.
```
[the remark this answers is not in the document]
Comment (gnl42) [reply mylyn.tasks/org.eclipse.mylyn.tasks.bugs/src/org/eclipse/mylyn/internal/tasks/bugs/wizards/SelectSupportElementPage.java:2]: Lovely :) So much to learn. 

What are 'red pipelines'?

[the remark this answers is not in the document]
Comment (ruspl-afed) [reply mylyn.tasks/org.eclipse.mylyn.tasks.bugs/src/org/eclipse/mylyn/internal/tasks/bugs/wizards/SelectSupportElementPage.java:2]: image

**Comment by ruspl-afed** [comment]
You have compilation problems after resolving deprecations

**Comment by gnl42** [comment]
I don't know what is going on. I have no errors when I build it locally (full test suite)

**Pull request merged**